### ISSUE TITLE
v0.3 release

### ISSUE THREAD

**Issue by adrinjalali** [skops-dev/skops#216]
I was wondering now that we have a persistence which we could ship, if we should do a release, it's been a while.

I would have liked to also include the new card implementation, but I think since that one doesn't change the API much from the users' perspective, we could include that in the next release, which we can do pretty quickly after v0.3. We probably would need to do another release soon with issues we find in the persistence.

WDYT @skops-dev/maintainers ?

**Comment by BenjaminBossan** [comment]
I agree that we can make a release now, there have been sufficiently many changes and we want users to test the new persistence feature. Let's not rush the new card implementation, I too think it can wait for the next release.
